apps/web-giddh/src/app/shared/template-froala/template-froala.component.ts

```
@@ -116,6 +116,10 @@ export class TemplateFroalaComponent implements OnInit {
     public bccEmails: any[] = [];
     /** Hold selected bcc email options */
     public selectedBccEmails: any[] = [];
+    /** Hold display to bcc and cc on click */
+    public showBccCc :Boolean = false;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)